src/parameters.ml

```diff
@@ -246,10 +246,7 @@ let[@inline] compute_current_protected_index (last_protected_index: Ligo.tez) (c
           )
        )
     )
-    (Ligo.mul_int_int
-       protected_index_inverse_epsilon
-       (Ligo.int_from_literal "1_000_000")
-    )
+    protected_index_inverse_epsilon
```

Comment:
Where did the extra `1_000_000` in the denominator go here? Was that just required when using `fraction_to_tez_floor`?

Reply:
Ah, that's a difference between `fraction_to_nat_floor` and `fraction_to_tez_floor`. `fraction_to_tez_floor` deals in `tez`, not `mutez`. So, for example
```ocaml
fraction_to_tez_floor (Ligo.int_from_literal "1_000_000") (Ligo.int_from_literal "1_000_000")
```
is equal to 1tez (which is 1_000_000mutez) and
```ocaml
fraction_to_tez_floor (Ligo.int_from_literal "1_234_567") (Ligo.int_from_literal "1_000_000")
```
is equal to `1_234_567mutez`, but
```ocaml
fraction_to_nat_floor (Ligo.int_from_literal "1_234_567") (Ligo.int_from_literal "1_000_000")
```
is simply equal to `1` (`= div 1234567 1000000`).

Reply:
Makes sense, thanks for the explanation!

Reply:
No prob! I do agree with @utdemir overall that scaling factors are a hassle and it would be nice to have less of them sprinkled over the codebase.